src/atoms/pill/component.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getClassName } from '@kibalabs/core';
+
+import { IComponentProps, defaultComponentProps, themeToCss, useBuiltTheme } from '../..';
+import { IPillTheme } from './theme';
+import { IIconProps, Spacing, PaddingSize } from '../../particles';
+
+interface IStyledPillProps {
+  theme: IPillTheme;
+}
+
+const StyledPill = styled.div<IStyledPillProps>`
+  ${(props: IStyledPillProps): string => themeToCss(props.theme.normal.default.text)};
+  ${(props: IStyledPillProps): string => themeToCss(props.theme.normal.default.background)};
+  outline: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  background-clip: border-box;
+  transition-duration: 0.3s;
+  &.fullWidth {
+    width: 100%;
+  }
+`;
+
+export interface IPillProps extends IComponentProps<IPillTheme> {
+  text: string;
+  isFullWidth: boolean;
+  iconRight?: React.ComponentClass<IIconProps>;
+  iconLeft?: React.ComponentClass<IIconProps>;
+  iconGutter?: PaddingSize;
+  onClicked?(): void;
+}
+
+export const Pill = (props: IPillProps): React.ReactElement => {
+  const theme = useBuiltTheme('pills', props.variant, props.theme);
+  return (
+    <StyledPill
+      id={props.id}
+      className={getClassName(Pill.displayName, props.className, props.isFullWidth && 'fullWidth')}
```

Comment:
fullWidth is not working for me for some reason. Despite having `fullWidth=false` the pill occupies the page width. Behaviour is different to button for example

Reply:
had to use `display:inline-flex` i think cos html `<button />` components have this by default but `<div />`s are `display: block` by default